### ISSUE TITLE
Datetime support

### ISSUE THREAD

**Issue by monitoth** [space10-community/conversational-form#67]
Do you plan to support in the future a datetime answer? For example, when you ask for the "Date of birth", in the answer the user should use a datetimepicker control. (Like bootstraps datetimepicker)

**Comment by felixnielsen** [comment]
@monitoth yes, this will be a part of the bigger rollout of version 1.0.0, stay tuned

**Comment by peimn** [comment]
http://felicegattuso.com/projects/datedropper/

http://felicegattuso.com/projects/timedropper/

These two awesome plugins can be integrated with CF

**Comment by danieltome** [comment]
Hi, just wanted to check if there was any progress with this?
I would love to help.

**Comment by berniebetgcarpio** [comment]
it seems like these 2 datepicker won't work?

http://felicegattuso.com/projects/datedropper/

http://felicegattuso.com/projects/timedropper/

**Comment by luc-7** [comment]
Hey! Any progress on this? Would be cool if you can update (us all)?

**Comment by elephantux** [comment]
@peimn how to integrate this datepicker with CF?

**Comment by jenssogaard** [comment]
**An update:** I do not think that date-time support will be part of the CF core at this time. However I have cooked up a crude example of how to implement [Datedropper3](https://felicegattuso.com/projects/datedropper/). Basicly, the approach is to hide the default textarea input and replace it with our custom input with a Datedropper3 instantiated on it. 

![Datedropper3 demo](http://cf.jenssogaard.com/screens/cf-date-picker.gif)

**Comment by winadv** [comment]
I am not able to insert Datedropper3 whenever error occurs. I would like an input help without using the code below:
`window.onload = function() {
  conversationalForm = window.cf.ConversationalForm.startTheConversation({
    formEl: document.querySelector("form"),
    context: document.querySelector("div.cf"),
    eventDispatcher: dispatcher
  });
};`

Because the same creates a kind of new form and that makes me make an error in mine.

**Comment by jenssogaard** [comment]
@winadv your problem is most likely that you have the `cf-form` attribute on your form-tag? so it instantiates twice.

I can not help you with your second question. You are better of using my example.

**Comment by winadv** [comment]
@jenssogaard Would you like to create an example using the script demo files, such as conditional-flow.html? I guarantee that with that I could adapt to my script.
Thanks

**Comment by jenssogaard** [comment]
@winadv it is kinda repeating what I have already done. Do you have issues implementing the example in your own project?

**Comment by aimer2695** [comment]
Hi.. there's a way to show the re captcha in the conversational form?..

**Comment by aimer2695** [comment]
Hi.. there's a way to show the re captcha in the conversational form?..

**Comment by aimer2695** [comment]
Hi.. there's a way to show the re captcha in the conversational form?..

**Comment by RachelGanonNew** [comment]
Hi ,
I'm using Angular Material and whould love to include their 'datepicker' in my cf form,
as for now it doent work for me though i have no errors
Any help will be very much appriacted!!
